lambdas.rst

```diff
@@ -10,7 +10,7 @@ normal functions and even behave like them.
 
 .. code:: python
 
-    lambda argument: manipulate(argument)
+    lambda arguments: expression
```

Comment:
IMO `manipulate(argument)` seems like a better choice. Perhaps you could revert this change and just go with the *ways* edit?